metadata-ingestion/docs/sources/mssql/mssql_pre.md

```diff
@@ -0,0 +1,10 @@
+### Prerequisites
+
+If you want to ingest MSSQL Jobs the user credentials needs the proper privileges.
+
+Script for granting the privileges:
+```
+USE MSDB
+GRANT SELECT ON OBJECT::msdb.dbo.sysjobsteps TO 'USERNAME'
+GRANT SELECT ON OBJECT::msdb.dbo.sysjobs TO 'USERNAME'
```

Comment:
I'm not really familiar with how mssql actually works, but it seems like there should also be some permissions required for stored procedures as well

I was looking at this https://www.sqlservercentral.com/forums/topic/permissions-to-view-stored-procedures

My goal here is mainly to save other folks this process of looking at our code + googling what `grant` commands need to be run

Reply:
Updated.  I`m not really familiar with mssql permission too :)  but I checked this script for new user and it works fine.

Reply:
Thanks!